rand-core/README.md

```diff
@@ -0,0 +1,21 @@
+rand-core
+====
+
+Core functionality for the [rand] library.
+
+This crate contains the core trait, `RngCore`, and some extension traits. This
+should be sufficient for libraries publishing an `RngCore` implementation,
+but most users should prefer to use the [rand] crate.
+Most parts of this crate are re-exported in [rand].
+
+[Documentation](https://docs.rs/rand-core)
+
+[rand]: ..
+
+
+# License
+
+`rand` is primarily distributed under the terms of both the MIT
+license and the Apache License (Version 2.0).
+
+See LICENSE-APACHE, and LICENSE-MIT for details.
```

Comment:
Should the license files (and the changelog) also be copied to the `rand-core` directory?

Reply:
I don't know, but that appears necessary to get the licence files inside the Cargo package, so I will add them.